tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -678,7 +696,7 @@ public function userHasCreatedTheFollowingLinkShare(string $user, TableNode  $bo
 	}
 
 	/**
-	 * @Given /^user "([^"]*)" has updated the last resource|space link share with$/
+	 * @Given user :user has updated the last resource/space link share with
```

Comment:
i don't think it matters much; either use regex or normal. While adding new step you could choose normal as per your need for you can skip refactoring old ones

Reply:
This was refactored because when i added the new step `And user "Alice" has updated the last resource share with:`, the step was navigating to two step definitions.